src/app/garfile/craft/index.njk

```diff
@@ -87,7 +87,7 @@
         <label class="govuk-label" for="craftReg">{{__('field_aircraft_registration')}}</label>
         {{ m.hint('craftReg', __('field_aircraft_registration_hint'))}}
         {{ m.error_message('craftReg', errors, __) }}
-        <input id="craftReg" name="craftReg" class="govuk-input govuk-input--width-10"
+        <input id="craftReg" name="craftReg" class="govuk-input govuk-input--width-10" autocomplete="off"
```

Comment:
Added these autocomplete="off" to try and prevent some sort of autocomplete capability.